_sections/30-projects.md

```diff
@@ -43,6 +43,7 @@ These projects pledge to drop Python 2 support in or before 2020.
 - [![](assets/osbrain.png)osBrain](https://github.com/opensistemas-hub/osbrain)
 - [![](assets/pymeasure.png)PyMeasure](https://github.com/ralph-group/pymeasure)
 - [![](assets/chaquopy.png)Chaquopy](https://chaquo.com/chaquopy/)
+- [![](assets/tryton.png)Tryton](http://www.tryton.org/)
```

Comment:
I think Tryton could probably be somewhere higher up the list - the idea is that projects are ordered by prominence. We haven't really got any good way to decide exactly where to put a project that's not on Github, though.